presidio-analyzer/analyzer/app_tracer.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+
+class AppTracer:
+    """This class handles app traces"""
+    def __init__(self, enable_interpretability=True):
+
+        logger = logging.getLogger('Interpretability')
+        if not logger.handlers:
+            ch = logging.StreamHandler()
+            formatter = logging.Formatter(
+                '[%(asctime)s][%(name)s][%(levelname)s]%(message)s')
+            ch.setFormatter(formatter)
+            logger.addHandler(ch)
+            logger.setLevel(logging.INFO)
+            logger.propagate = False
+
+        self.logger = logger
+        self.enable_interpretability = enable_interpretability
+
+    def trace(self, request_id, trace_data):
+        """
+        Writes interpretability trace
```

Comment:
Here too. Assume a new developer doesn't know what we mean by 'interpretability'

Reply:
I am not sure I understand.
Is it a semantic request? I can leave out the word "interpretability"

Reply:
The entire documentation of this class is very limited. We want a newcomer to understand why we have both Logger and AppTracer, and how the trace method should be used. It's ok to have the word "interpretability", but it should be much more detailed. For example:
```
class AppTracer:
    """
    This class provides the ability to log/trace the system's decisions, 
    such as which modules were used for detection, 
    which logic was utilized, what results were given and potentially why. 
    This can be useful for analyzing the detection accuracy of the system.
    """
```

```
def trace(self, request_id, trace_data):
       """
       Writes a value associated with a decision for a specific request into the trace, 
       for further inspection if needed.
       """
```